javalib/src/main/scala/java/time/MonthDay.scala

```diff
@@ -0,0 +1,160 @@
+package java.time
+
+import java.time.chrono.IsoChronology
+
+import scala.scalajs.js
+
+import java.time.temporal._
+
+/**
+  * Created by alonsodomin on 22/12/2015.
+  */
```

Comment:
Oh, I missed this formatting issue with comments, we align to the first `*` and start in the first line, i.e.:
```scala
/** Created by alonsodomin on 22/12/2015.
 */
```